Public/Src/Engine/Processes/ReportedFileOperation.cs

```diff
@@ -172,6 +172,13 @@ public enum ReportedFileOperation : byte
         /// </summary>
         ChangedReadWriteToReadAccess,
 
+        /// <summary>
+        /// This is a quazy operation. We issue this only when <see cref="FileAccessPolicy.OverrideAllowWriteForExistingFiles"/> is set, representing

```

Comment:
What's "quazy"?

Reply:
'quasi'. Copy paste from the value above

---
In reply to: [299159736](https://github.com/microsoft/BuildXL/pull/536#discussion_r299159736) [](ancestors = 299159736)